--- a/bionic/libc/arch-arm/cortex-a9/bionic/memcpy_base.S
+++ b/bionic/libc/arch-arm/cortex-a9/bionic/memcpy_base.S
@@ -44,7 +44,7 @@
         /* check if buffers are aligned. If so, run arm-only version */
         eor         r3, r0, r1
         ands        r3, r3, #0x3
-        beq         __memcpy_base_aligned
+        beq         MEMCPY_BASE_ALIGNED
 
         /* Check the upper size limit for Neon unaligned memory access in memcpy */
         cmp         r2, #224
@@ -220,4 +220,4 @@
 6:
         ldmfd       sp!, {r4-r8}
         ldmfd       sp!, {r0, pc}
-END(MEMCPY_BASE_ALIGNED)
+END(MEMCPY_BASE_ALIGNED)
\ No newline at end of file
